remotemonitor/monitor.go

```diff
@@ -126,7 +126,11 @@ func (m *Monitor) reportErr(i Instance, err error, action string) {
 			log.Println("No ErrorAPIKey for", ins.Location)
 			continue
 		}
-		go ins.createGenericAlert(ins.ErrorAPIKey, "", summary, details)
+		go func(ins Instance, summary string, details string) {
+			if err := ins.createGenericAlert(ins.ErrorAPIKey, "", summary, details); err != nil {
+				log.Printf("ERROR: create generic alert: %v", err)
+			}
+		}(ins, summary, details)
```

Comment:
The summary and details are outside the loop and don't get reassigned, so having them passed here isn't required. 

Additionally, using `ins := ins` at the top of the loop body has the benefit of being less noisy and making the type inference for us.

```suggestion
		ins := ins // copy
		go func() {
			if err := ins.createGenericAlert(ins.ErrorAPIKey, "", summary, details); err != nil {
				log.Printf("ERROR: create generic alert: %v", err)
			}
		}()
```


https://github.com/golang/go/discussions/56010#discussioncomment-3813292